{"name": "test_add[小数]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 34.066700000000004 == 34.0667\n  +34.066700000000004\n  -34.0667", "trace": "self = <testing.testFixture.test_operation2.TestB object at 0x000001F1790F9EE0>\nx = 34.002, y = 0.0647, expected = 34.0667\n\n    @pytest.mark.run(order=1)\n    @pytest.mark.parametrize('x, y, expected', yaml_data_with_key('valid_add')[0],\n                             ids=yaml_data_with_key('valid_add')[1])\n    def test_add(self, x, y, expected):\n        r = add(x, y)\n        print(f\"{x}+{y}=\", r)\n>       assert r == expected\nE       assert 34.066700000000004 == 34.0667\nE         +34.066700000000004\nE         -34.0667\n\ntest_operation2.py:19: AssertionError"}, "attachments": [{"name": "stdout", "source": "b6211577-0e65-45c9-8846-98732542998b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "34.002"}, {"name": "y", "value": "0.0647"}, {"name": "expected", "value": "34.0667"}], "start": 1597887873838, "stop": 1597887873840, "uuid": "c4409176-4d0c-40c5-80e3-d22f81a265fa", "historyId": "884d0042478ce32ad1edd54117f0960e", "testCaseId": "70ab598a7d25549ff126384532e89726", "fullName": "testing.testFixture.test_operation2.TestB#test_add", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "testing.testFixture"}, {"name": "suite", "value": "test_operation2"}, {"name": "subSuite", "value": "TestB"}, {"name": "host", "value": "LAPTOP-O258QM4M"}, {"name": "thread", "value": "6460-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testing.testFixture.test_operation2"}]}